index.js

```diff
@@ -89,11 +90,29 @@ const getCompiledFileSync = (moduleDir, path) => {
   return transformSrc(moduleDir, res.toString())
 }
 
+const getBsConfigModuleOptions = buildDir => {
```

Comment:
Can this be async instead of blocking? The loader is run for everyfile and this would be a lot of overhead